docs/connection.rst

```diff
@@ -91,9 +91,11 @@ Developers can register custom classes to handle driver types using the `Conneci
    Connection::register('mysql', Custom\MySQL\Connection::class); // or directly using the class
    Custom\MySQL\Connection::register();
    
-The driver type used in the latter case is the default value of the `$driverType` property of `Custom\MySQL\Connection`
+The driver type used in the latter case is the default value of the `$driverType` property of 
+`Custom\MySQL\Connection`
 
-.. php:method:: createDriverConnection(array $dsn)
+.. php:method:: establishDriverConnection(array $dsn)
```

Comment:
isn't `establish` a `connect`?

Reply:
Yes, it is but Connection::establish is semantic

Reply:
`establish` is the same as `connect` to me, but `connect` sounds like simple english...

Anyway, I do not like `xxxDriverConnection` because it returns `Connection` and not the driver...

Reply:
It returns driver connection as it says in the name. Not a driver.
As I said connections in dsql are nested so this is a way to distinguish the nested connection which "drives" the parent.

Reply:
I see - driver vs. connection. We should not mix it up, so probably only `establishDriver` here.

But still not sure if `establish` is the right term... I am not amazed with it, but not strictly negative...

Reply:
since class is called Connection, calling method connect seems redundant. OK to rename into establish.